pkg/network/protocols/http2/model_linux.go

```diff
@@ -134,16 +134,56 @@ func (tx *EbpfTx) ConnTuple() types.ConnectionKey {
 	}
 }
 
+// stringToHTTPMethod converts a string to an HTTP method.
+func stringToHTTPMethod(method string) (http.Method, error) {
+	switch strings.ToUpper(method) {
+	case "PUT":
+		return http.MethodPut, nil
+	case "DELETE":
+		return http.MethodDelete, nil
+	case "HEAD":
+		return http.MethodHead, nil
+	case "OPTIONS":
+		return http.MethodOptions, nil
+	case "PATCH":
+		return http.MethodPatch, nil
+	// Add more cases for other HTTP methods as needed
+	default:
+		return 0, fmt.Errorf("unsupported HTTP method: %s", method)
+	}
```

Comment:
take a look in the definition of the Method* consts
eventually something like:
```go
switch vaL := strings.ToUpper(method):
case http.MethodPut, http.MethodDelete,...:
    return val, nil
default:
....
```

Reply:
also, let's add support for post and get, as there is no denial for such use-case to happen